ui/app/src/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
     * `LegacyRecentActivityDashlet`
     * `LegacyRecentlyPublishedDashlet`
     * `LegacyUnpublishedDashlet`
+  * `WorkflowCancellationDialogUI`: prop `items` type changed from an async Resource to sync SandboxItem array.
```

Comment:
The format for 4.2.0 changes is a bit different, so I put the component name only, under the components section